### PR TITLE
Update NEWS for 1.11

### PR DESCRIPTION
**Pull request by fatteneder**

### CODE REVIEW

**Comment by Seelengrab** [comment]
Please provide a descriptive PR title & description.

File: NEWS.md [hunk 1]
```diff
@@ -113,6 +117,8 @@ New library features
 * `filter` can now act on a `NamedTuple` ([#50795]).
 * `Iterators.cycle(iter, n)` runs over `iter` a fixed number of times, instead of forever ([#47354]).
 * `zero(::AbstractArray)` now applies recursively, so `zero([[1,2],[3,4,5]])` now produces the additive identity `[[0,0],[0,0,0]]` rather than erroring ([#38064]).
+* `include_dependency(path; track_content=true)` allows to switch from using `mtime`
```
Comment (giordano) [inline NEWS.md:120]:
```suggestion
* `include_dependency(path; track_content=true)` allows switching from using `mtime`
```

File: NEWS.md [hunk 2]
```diff
@@ -35,6 +35,8 @@ Language changes
   execution ([#52123]).
 * Specifying a path in `JULIA_DEPOT_PATH` now results in the expansion of empty strings to
   omit the default user depot ([#51448]).
+* Precompilation cache files are now relocatable and their validity is now verified through
+  a content hash of the `include` files instead of their `mtime` ([#49866]).
```
Comment (JeffBezanson) [inline NEWS.md:39]:
```suggestion
  a content hash of their source files instead of their `mtime` ([#49866]).
```

File: NEWS.md [hunk 3]
```diff
@@ -82,6 +84,8 @@ New library functions
 * `copyuntil(out, io, delim)` and `copyline(out, io)` copy data into an `out::IO` stream ([#48273]).
 * `eachrsplit(string, pattern)` iterates split substrings right to left ([#51646]).
 * `Sys.username()` can be used to return the current user's username ([#51897]).
+* `Sys.isreadable(), Sys.iswriteable` can be used to check if the current user has access permissions
```
Comment (lgoettgens) [inline NEWS.md:87]:
```suggestion
* `Sys.isreadable(), Sys.iswriteable()` can be used to check if the current user has access permissions
```

File: NEWS.md [hunk 4]
```diff
@@ -82,6 +84,8 @@ New library functions
 * `copyuntil(out, io, delim)` and `copyline(out, io)` copy data into an `out::IO` stream ([#48273]).
 * `eachrsplit(string, pattern)` iterates split substrings right to left ([#51646]).
 * `Sys.username()` can be used to return the current user's username ([#51897]).
+* `Sys.isreadable(), Sys.iswriteable()` can be used to check if the current user has access permissions
```
Comment (fatteneder) [inline NEWS.md:87]:
```suggestion
* `Sys.isreadable(), Sys.iswritable()` can be used to check if the current user has access permissions
```

Fixing spelling in accordance with #53699

**Pull request merged**